tst/EnergyPlus/unit/HVACUnitaryBypassVAV.unit.cc

```diff
@@ -68,6 +69,8 @@ TEST_F(EnergyPlusFixture, UnitaryBypassVAV_GetInputZoneEquipment)
 
     std::string const idf_objects = delimited_string({
 
+        "Zone,",
+        "  Zone 2;                                 !- Name",
```

Comment:
Add another zone so "Zone 1" is the second zone in the Zone() and CurDeadBandOrSetback() arrays.